ci/ubuntu-16.04.docker

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:16.04
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+&& apt-get install -y \
+      git \
+      cmake \
+      make \
+      g++ \
+      libclang-3.5-dev \
+      liblldb-3.5-dev \
+      clang-format-3.5 \
```

Comment:
Can use the libclang/liblldb/clang-format 3.6 version instead. Currently this is the default llvm version in debian testing/unstable.